analyzers/tests/SonarAnalyzer.Test/SymbolicExecution/Roslyn/LoopDetectorTest.cs

```diff
@@ -312,8 +312,8 @@ public class LoopDetectorTest
             }
             _ = "After loop";                // Block 7
             """,
-            [2, 3, 4, 5],
-            [1, 6, 7]);
+            [2, 3, 4, 5, 6],
```

Comment:
This change will again need more testing. Especially around the `Where` condition